### PR TITLE
Remove unneeded godot-cpp build step from the GitHub action.

### PR DESCRIPTION
**Pull request by Ivorforce**
Briefly discussed on Rocket Chat, it seems this is wholly just not needed for the runner, as godot-cpp is built as part of the GDExtension build step.

I know some repositories suggest building the deps first and the project later, using separate caches for each such that a rebuild can go faster. But as far as I've seen, in this project it doesn't work because the later build step just re-builds godot-cpp from scratch anyhow (at least for me). If anybody's seen evidence to the contrary, please share them.

### CODE REVIEW

**Comment by Ivorforce** [comment]
Paging @Ughuuu because this PR would remove the option d100b72a3a1017ce2bebbe2d23b4c0fab6158a92 just added.

**Review by paddy-exe — APPROVED** [review]

**Comment by paddy-exe** [comment]
Looks good to me. However will wait for others to approve as well before merging.

**Comment by Ivorforce** [comment]
Yes, we should definitely have someone else test the builds on this branch before merging.

**Comment by Ughuuu** [comment]
Looks good, will remove then in my build again, np.

**Review by Ughuuu — APPROVED** [review]

**Comment by paddy-exe** [comment]
Thanks for the quick PR @Ivorforce

**Pull request merged**
